packages/svelte/src/compiler/phases/3-transform/server/visitors/CallExpression.js

```diff
@@ -51,7 +50,23 @@ export function CallExpression(node, context) {
 	}
 
 	if (rune === '$inspect' || rune === '$inspect().with') {
-		return transform_inspect_rune(node, context);
+		if (!dev) return b.empty;
+
+		const call =
+			rune === '$inspect'
+				? node
+				: /** @type {CallExpression} */ (/** @type {MemberExpression} */ (node.callee).object);
+
+		const args = call.arguments.map((arg) => /** @type {Expression} */ (context.visit(arg)));
+
+		const inspector =
+			rune === '$inspect'
+				? 'console.log'
+				: /** @type {Expression} */ (context.visit(node.arguments[0]));
```

Comment:
It might be silly for `$$args` but this is a lot of specific and repeated code...can we have a little utility to do this work?